exmps.py

```diff
@@ -0,0 +1,15 @@
+color_list = ["Red", "Green", "White", "Black"]
```

Comment:
What is the purpose of this program?
Suggestion: Write comments for complex programs to explain the problem statement